e2e/steps/RequestToken.mjs

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable new-cap */
+import { Then } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import { fixture } from '../fixtures/page.mjs';
+
+Then(
+  '{string} should be selected in {string} dropdown',
+  async function (selectedText, dropdownTitle) {
+    const title = fixture.page
+      .getByText(dropdownTitle, { exact: true })
+      .locator('..')
+      .getByTestId('selected-menu-item')
+      .getByText(selectedText, { exact: true });
+    const content = await title.textContent();
+    expect(content).toBeTruthy();
```

Comment:
I think this should be in common since it might be used by some other test in the feature

Reply:
Fixed